src/features/location/reducers.js

```diff
@@ -0,0 +1,18 @@
+import { LOCATION_TYPES } from './types';
+
+export default (state = {}, action) => {
+  switch (action.type) {
+    case LOCATION_TYPES.FETCH_SUCCEEDED:
+      return {
+        ...state,
+        locationTags: action.locationTags,
```

Comment:
I would just call this "locations", because I believe this is where you are storing all the locations returned by the REST call... (looks like this is what they called it in the Order Entry module but but it's not really correct... location tags are different than locations...)